src/file/file.service.ts

```diff
@@ -121,6 +152,10 @@ export class FileService {
     return { succeeded: deletedFiles, allSucceeded: pathSuccess };
   }
 
+  static async isShortcut(file: any): Promise<boolean> {
+    return file.type === shortcutModelName;
+  }
```

Comment:
You should return `isShortcurt` property on every returned file.
You don't even use this function...

Reply:
Nevertheless, I don't think this function will work without changes.

Reply:
i changed the code in the function, but i can't find a place where i need to use.
i remember in each scenario i needed to find which type of file it is (in the repo) i found a solution with you.
idk.

Reply:
did everything we talked about, done.